site/guide/working-with-model-inventory.qmd

```diff
@@ -0,0 +1,51 @@
+---
+title: "Working with the model inventory"
+date: last-modified
+listing:
+  - id: model-inventory-listing
+    type: grid
+    max-description-length: 250
+    sort: false
+    fields: [title, description]
+    contents:
+      - register-models-in-model-inventory.qmd
+      - edit-model-inventory-fields.qmd
+      
+---
+The model inventory holds all models. Within the model inventory, register new models and edit fields. Organize the inventory by searching, filtering and sorting models. Configure the inventory by managing column layout and choosing card or table view.
+
+## Register models and edit fields
+See these guides on how to add new models and edit their fields.
+
+:::{#model-inventory-listing}
+:::
+
+## Search, filter, and sort models
```

Comment:
- I don't think search, filter or sort need gifs or really even images at all. But if you want images here, I would recommend a static cap like so: http://localhost:3663/guide/work-with-model-findings.html#filter-project-findings
- Also, how do I GET to the inventory to search, filter, sort? You forgot the step-by-step!

Reply:
I agree, I must let go of the gifs lol. Should I just do a link to the model inventory as a step? It would bypass the `Log in to the Platform UI` step too!

Reply:
I would remain consistent with the other step-by-steps on the docs site. The logging in part is an important step. ;) People can skip if it they know they're logged in, but the doc's job is to break it down as granular as possible (so that images are only supplementary, not doing the heavy lifting!).